x-pack/plugins/actions/server/lib/task_runner_factory.test.ts

```diff
@@ -777,6 +778,7 @@ test(`treats errors as errors if the task is retryable`, async () => {
   expect(err instanceof ExecutorError).toEqual(true);
   expect(err.data).toEqual({ foo: true });
   expect(err.retry).toEqual(false);
+  expect(isRetryError(err)).toEqual(false);
```

Comment:
should we update the test name? it is called `will throw a retry error` but `isRetryError` should return `false`?

Reply:
Resolved in this commit: https://github.com/elastic/kibana/commit/3f5fbd14c06b90082cad8c953e018a16213cf615